codejail/proxy.py

```diff
@@ -17,8 +18,8 @@
 # communicate a few values, and unpack them.  Lastly, we need to be sure we can
 # handle binary data.  Serializing with repr() and deserializing the literals
 # that result give us all the properties we need.
-serialize = repr
-deserialize = ast.literal_eval
+SERIALIZE = repr
+DESERIALIZE = ast.literal_eval
```

Comment:
All functions are constants, I don't see a reason to spell them as if they were numeric constant.